### ISSUE TITLE
Ensure point sizing behavior is the same for matplotlib and bokeh

### ISSUE THREAD

**Issue by philippjfr** [ioam/holoviews#831]
It seems Points/Scatter size scaling is inconsistent between matplotlib and bokeh. In matplotlib the size of the points is set via the ``s`` style options while bokeh uses ``size``. Additionally it seems point scaling in matplotlib produces significantly larger points. Both issues should be addressed.

**Comment by jbednar** [comment]
Fixing this would help avoid a lot of confusion, because matplotlib accepts ``size`` but ignores it.

**Comment by jbednar** [comment]
This just bit me *again* today, in part because [our own documentation](http://holoviews.org/reference/elements/matplotlib/Points.html) shows using "size", which appears to be incorrect for matplotlib. I say above that mpl ignores "size", but that appears to be because I'm generally loading a bokeh backend also whenever I use mpl, and so it accepts the bokeh keyword without complaint and ignores it?  That doesn't explain why the documentation doesn't show errors for "size" for the matplotlib backend, though.  We at least need to fix our docs, but also maybe there is better error checking to do when building the docs?

**Comment by jlstevens** [comment]
I have also been bitten by this multiple times i.e ``s`` vs ``size``. I don't get why matplotlib ignores the latter, shouldn't ``s`` be a short alias for ``size`` which should also work?

**Comment by philippjfr** [comment]
Right, it does bite me frequently too. I'd be in favor of translating some common options between matplotlib and bokeh, including ``s`` and ``size`` but also things like ``line_width``/``linewidth``, ``line_color``/``edge_color``, ``fill_color``/``facecolor`` and a few others.

**Comment by jlstevens** [comment]
I have mixed feelings about that, as I do feel things could be easier but it would also break our story about holoviews passing style options straight to the backend. 

Adding some simple aliases would certainly be convenient but I'm not sure where to draw the line - we don't want to be in the business of trying to make different plotting extensions consistent (either in API or output) as that task is both without end and most likely impossible.

**Comment by jbednar** [comment]
I can see lots of arguments in many directions pro and con about such aliases.  Maybe we could enable them in hv.extension automatically by default when loading multiple backends, but allow the aliases to be enabled explicitly in other cases and allow them to be explicitly disabled if not desired?
